packages/backend/src/database/entities/userAttributes.ts

```diff
@@ -6,7 +6,7 @@ export type DbUserAttribute = {
     name: string;
     description?: string;
     organization_id: number;
-    default: string | null;
+    attributeDefault: string | null;
```

Comment:
this should be `attribute_default` right?

Reply:
Updated and tested 

![Screenshot from 2023-09-07 16-02-57](https://github.com/lightdash/lightdash/assets/1983672/67592a77-ac48-48f1-be01-7af31b49a5bf)
![Screenshot from 2023-09-07 16-03-06](https://github.com/lightdash/lightdash/assets/1983672/dd291437-efff-4be9-9dc0-715faafe55aa)